automated/health-checks/health-check_RevPi_Connect_001.yaml

```diff
@@ -0,0 +1,41 @@
+device_type: RevPi_Connect
+job_name: RevPi_Connect - Standard health check 001 
+priority: medium
+visibility: public
+
+timeouts:
+  job:
+    minutes: 5
+  action:
+    minutes: 1
+  connection:
+    minutes: 2
+    
+actions:
+- deploy:
+    timeout:
+      minutes: 4
+    to: ssh
+- boot:
+    method: ssh
+    connection: ssh
+    prompts: ["root@RevPi"]
+    timeout:
+      minutes: 2
+
+- test:
+    timeout:
+      minutes: 5
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: worker-device-test
+          description: "Simple test if the test will be executed on given devices on a specific worker"
+        run:
+          steps:
+          - lava-test-case test1-pass --result pass
+          - lava-test-case test2-pass --result pass
```

Comment:
`lava-test-case` seems to be a LAVA helper script. The documentation is honestly so hard to read that I'm not sure what it actually does.
Where does the test `test1-pass` come from? Same with `test2-pass`. Is it just an arbitrary name of the test?
Is this "test" just 2 calls to LAVA to tell it that essentially nothing passed? What is achieved with this exactly? A comment would be helpful here.

Reply:
There are different types of helpers.
They interact with the dispacher and can indicate different states in the tests, being able to show information or directly canceling the test completely.
In this case "lava-test-case" is used to indicate the status of a test. The name under which the information is displayed is passed as the first parameter, in this case it is "test1-pass". Another parameter used is "--result" which can be "pass" or "fail".
Basically what this does is to indicate WITHOUT CANCELING THE JOB that the test does not complete the necessary conditions. In the WebInterface, this is shown with a red or green box.

The file "health-check_RevPi_Connect_001.yaml" and "health-check_RevPi_Core_001.yaml" are old and do not check anything. They only indicated that the ssh connection still exists, indicating that the device is still connected. 
If the device is not available, LAVA will show them as OFFLINE and nothing else can be run on them until someone changes their status.
This way, debugging can be performed better.

I have added them to have a better tracking of changes.

Reply:
I see, thank your for explaining. This motivation behind these tests should be documented here, even if these are deprecated or similar. Also, they should ideally have different names to show what they are for.